articles/openshift/oshift-how-restrict-access-to-openshift-routes-by-ip.md

```diff
@@ -0,0 +1,57 @@
+---
+title: UKCloud Knowledge Centre article about whitelisting public IP's on your openshift routes | UKCloud Ltd
```

Comment:
Change this to match to actual title of the article:

How to restrict access to a route in openshift by IP[s] address